ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py

```diff
@@ -212,3 +213,29 @@ def test_paddle_equal_all(
         x=x[0],
         y=x[1],
     )
+
+
+# logical_or
+@handle_frontend_test(
+    fn_tree="paddle.logical_or",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=("bool",), num_arrays=2),
```

Comment:
Our policy is to set `available_dtypes` to "valid" and refine the supported dtypes in the function's decorator.